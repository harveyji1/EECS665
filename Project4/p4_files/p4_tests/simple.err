/bin/sh: 1: ../dmc: not found
