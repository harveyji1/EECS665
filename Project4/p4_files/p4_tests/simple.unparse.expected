glob{bool} : bool;
main{()->int} : (){
	a{int} : int;
	glob{bool} = a{int};
}
